keyboards/kepler_33/readme.md

```diff
@@ -0,0 +1,26 @@
+# Kepler-33
+
+![Kepler-33](https://i.imgur.com/zVvzBrO.png)
```

Comment:
This uses a smaller, but still high res version of the image, that is more suitable for github markdown rendering. 
```suggestion
![Kepler-33](https://i.imgur.com/zVvzBrOl.png)
```